mm2src/coins/lp_coins.rs

```diff
@@ -2085,7 +2085,8 @@ pub trait MarketCoinOps {
     /// Signs raw utxo transaction in hexadecimal format as input and returns signed transaction in hexadecimal format
     async fn sign_raw_tx(&self, args: &SignRawTransactionRequest) -> RawTransactionResult;
 
-    fn wait_for_confirmations(&self, input: ConfirmPaymentInput) -> Box<dyn Future<Item = (), Error = String> + Send>;
+    /// Wait for confirmations and return confirmation number
+    fn wait_for_confirmations(&self, input: ConfirmPaymentInput) -> Box<dyn Future<Item = u64, Error = String> + Send>;
```

Comment:
> and return confirmation number

I guess you mean the number of confirmed blocks?

Reply:
Yes, I thought that "confirmation number" is a synonymous for the "number of confirmed blocks" that have been added to the blockchain after a transaction is included in a block. But may be doc comm is not clear.

Fixed doc com https://github.com/KomodoPlatform/komodo-defi-framework/pull/2199/commits/b50b2a3f33201d980d33c74b9c225095ca3cbf5e

Hmm yeah, "confirmation number" is not correct, "confirmations number" is better. anyway I changed it to "number of confirmed blocks"